components/org.wso2.carbon.identity.conditional.auth.functions.user/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/user/Utils.java

```diff
@@ -148,4 +149,60 @@ public static String getUserIdClaimURI(String federatedIdpName, String tenantDom
         return userIdClaimURI;
     }
 
+    /**
+     * Retrieve the groups claim configured for the federated IDP.
+     *
+     * @param federatedIdpName  Federated IDP name.
+     * @param tenantDomain      Tenant domain.
+     * @return  groups claim configured for the IDP.
+     * @throws IdentityProviderManagementException
+     */
+    public static String getGroupsClaimURIByClaimMappings(String federatedIdpName, String tenantDomain) throws
```

Comment:
Can't this be a private method in the class that calls it?
Will this be useful or used by other functions?

Reply:
Resolved by https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/132/commits/97d487ff047a845a9b889b5d656e09290439ba0f